#!/usr/bin/python3
def raise_exception():
    a = 5
    b = "a"
    sum = 0
    sum = a / b
    return sum